source/color_yuv420_to_rgb.ipynb

```diff
@@ -0,0 +1,325 @@
+{
```

Comment:
<p>%%capture</p><p>!pip install --upgrade git+https://github.com/kornia/kornia.git</p><p>!pip install py7zr</p><p> </p><p> </p><p> </p><p>this will remove the stdout</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kornia/tutorials/pull/24/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>